Libraries/LLM/Tokenizer.swift

```diff
@@ -67,54 +67,13 @@ public func loadTokenizer(configuration: ModelConfiguration) async throws -> Tok
         tokenizerConfig = Config(dictionary)
     }
 
-    // workaround: some merges can't be split on space in BPETokenizer
-    if let tokenizerClass = tokenizerConfig.tokenizerClass?.stringValue {
-        switch tokenizerClass {
-        case "T5Tokenizer":
-            break
-        default:
-            tokenizerData = discardUnhandledMerges(tokenizerData: tokenizerData)
-        }
-    }
```

Comment:
This is fixed in:

- https://github.com/huggingface/swift-transformers/issues/51
- https://github.com/huggingface/swift-transformers/pull/52

Removing workaround